website/blog/2023-11-15-run-webassembly-wasm-workloads-windows-and-macos.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Use WebAssembly on macOS and Windows
```

Comment:
Remove 'Use', or make the title more flashy.

Reply:
I had a flashy title but then it was not contained in the sidebar (was too long)
so I don't know if we can have a short & flashy title ? (or a long title but it'll wrap the title in the sidebar)

Reply:
Yeah, understood. I guess it's fine to keep it short for sidebar, but I'd still remove 'Use' as it doesn't really add anything and feels like a filler word.

Reply:
"Unlock the Power of WebAssembly on macOS and Windows" is more punchy but too long 
"Harness WebAssembly: macOS & Windows" or "WebAssembly: Mac & Windows Mastery"

Reply:
I like the first but agree about length, and 'mastery' sounds a bit ambitious. So +1 to any like this:
"Unlock WebAssembly on macOS & Windows"
"Harness WebAssembly on macOS & Windows"
"Exploit WebAssembly on macOS & Windows"

Reply:
took the first one